### PR TITLE
Create the alphaMountain package

### PR DESCRIPTION
**Pull request by dominiklohmann**

### CODE REVIEW

File: alphamountain/package.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+author_icon: https://github.com/tenzir.png
+package_icon: |
+  https://raw.githubusercontent.com/tenzir/library/main/alphamountain/package.png
+
+description: |
+  [alphaMountain](https://alphamountain.ai) provides up-to-date domain and IP
+  intelligence for cybersecurity investigational and protection platforms.  
+
+  This package provides the `alphamountain-threats` context for enrichment and
+  pipelines that constantly update the feed.
```
Comment (mavam) [inline alphamountain/package.yaml:13]:
```suggestion
  pipelines that periodically update the feed.
```

File: alphamountain/package.yaml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+inputs:
+  license:
+    name: License key
+    description: |
+      Your alphaMountain batch API key.
+    type: string
+    default: 00000000-0000-0000-0000-000000000000
```
Comment (mavam) [inline alphamountain/package.yaml:21]: Is this a sane default? Or no default at all?

File: alphamountain/package.yaml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    description: |
+      A pipeline that updates the `alphamountain-threats` context.
+    definition: |
+      // tql2
+      | subscribe "alphamountain"
+      | where @name == "alphamountain.threat"
+      | legacy "context update alphamountain-threats --key=url --create-timeout {{ inputs.threat-expiry }}"
+
+snippets:
+  - name: Match all string and IP fields
```
Comment (mavam) [inline alphamountain/package.yaml:85]: Does that work on both strings and IPs? How?

**Review by mavam — APPROVED** [review]
Nice addition!

File: alphamountain/package.yaml (hunk 2 above)
Comment (dominiklohmann) [reply alphamountain/package.yaml:21]: Ideally this would be a hint, but this is better than nothing.

File: alphamountain/package.yaml (hunk 3 above)
Comment (dominiklohmann) [reply alphamountain/package.yaml:85]: Yes, through magic of the `lookup` operator.

**Pull request merged**